customtop10/functions.js

```diff
@@ -114,7 +114,7 @@ function highlight_code() {
 function top_10_code() {
     var total_entries = get_entry_count();
     if (total_entries == 0) return false;
-    var entries = "398000" + pad(total_entries, 2);
+    var entries = "3980000" + total_entries.toString(16).toUpperCase();
```

Comment:
Not necessary imo, also the entire code is set to touppercase at the very end

Reply:
could just remove the method call, but nonetheless you don't need to pad it at all :P

Reply:
Sure, I just think the function call looks cleaner